### PR TITLE
Cmake fix

### PR DESCRIPTION
**Pull request by ahundt**
Identical to https://github.com/jrl-umi3218/mc_rbdyn_urdf/pull/7 but also actually updates the cmake subrepository so it should be ready to merge now.

### CODE REVIEW

**Comment by ahundt** [comment]
@gergondet any chance you could give this a quick look?

File: CMakeLists.txt [hunk 1]
```diff
@@ -28,6 +28,7 @@ set(PROJECT_URL "https://github.com/jrl-umi3218/mc_rbdyn_urdf")
 set(PROJECT_DEBUG_POSTFIX "_d")
 
 set(DOXYGEN_USE_MATHJAX "YES")
+SET(CXX_DISABLE_WERROR True)
```
Comment (gergondet) [inline CMakeLists.txt]: Please use lower-case `set` here

File: CMakeLists.txt [hunk 2]
```diff
@@ -61,5 +62,6 @@ if(${PYTHON_BINDING})
 endif()
 
 pkg_config_append_libs(mc_rbdyn_urdf)
-
 setup_project_finalize()
+SETUP_PROJECT_PACKAGE_FINALIZE()
```
Comment (gergondet) [inline CMakeLists.txt]: Please use lower-case `setup_project_package_finalize()` here

**Review by gergondet — CHANGES_REQUESTED** [review]
Thanks for the PR.

Will merge when these 2 minor issues are addressed.

**Comment by gergondet** [comment]
Hi @ahundt, thanks to you and @Chunting for this PR.

I will merge once these two small issues are fixed.

Does this mean we can close #7?

**Comment by ahundt** [comment]
@gergondet should be fixed and yes #7 can be closed

**Review by gergondet — APPROVED** [review]

**Comment by gergondet** [comment]
Thanks!

Merging this and closing #7

**Pull request merged**